packages/server/test/adapters/aws-lambda.test.tsx

```diff
@@ -0,0 +1,121 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { z } from 'zod';
+import * as trpc from '../../src';
+import { inferAsyncReturnType } from '../../src';
+import * as trpcLambda from '../../src/adapters/aws-lambda';
+import { mockAPIGatewayProxyEvent } from './aws-lambda.utils';
+
+const createContext = async ({
+  req,
+}: trpcLambda.CreateLambdaContextOptions) => {
+  return {
+    user: req.headers['X-USER'],
+  };
+};
```

Comment:
as @mmkal suggested, I agree this should become

```suggestion
const createContext = async ({
  event,
}: trpcLambda.CreateLambdaContextOptions) => {
  return {
    user: event.headers['X-USER'],
  };
};
```